lib/s3/object.rb

```diff
@@ -84,7 +84,7 @@ def request_acl
     # Downloads the content of the object, and caches it. Pass true to
     # clear the cache and download the object again.
     def content(reload = false)
-      return @content if defined?(@content) and not reload
+      return @content unless reload or @content.nil?
```

Comment:
There is a logic change here to (hopefully) improve readability since `if not x and not y` is a little hard to read. `unless x or y` is a bit more straightforward.